textfinding_indikator4.py

```diff
@@ -0,0 +1,154 @@
+# this code is for TEXT FINDING - INDIKATOR 4
+# input is txt file from preprocess dokumen lama, nama instansi & judul from dokumen lama & baru
+
+import re
+import preprocess_dokbaru as dokbaru
+import preprocess_doklama as doklama
+from utility import *
+
+
+def txtreader(filename, lv, keyword):
+    # func to search keyword in txt file
+
+    # open txt file
+    file = open(f'cleaned_{filename}.txt', 'r')
+
+    idx = 0
+    result = []
+
+    # read line by line from txt
+    for line in file:
+
+        if (lv == 1):
+            reg = f'(?:(pembangunan)(.*?)(aplikasi)\s+(spbe)?)'
+            if (re.search(reg, line, re.IGNORECASE)):
+                result.append([idx, line])
+
+        elif (lv == 2):
+            for key in list(keyword):  # create a duplicate list
+                if key in keyword:  # check if element in iteration exist in otiginal list, skip element if not exist
+                    if re.search(key, line, re.IGNORECASE):
+
+                        # to include ['aplikasi', 'aplikasi khusus', 'aplikasi umum', 'aplikasi spbe']
+                        reg = f'(?:(aplikasi)\s+[(umum)|(khusus)|(spbe)]?)'
+                        if re.search(reg, line, re.IGNORECASE):
+
+                            # exclude words
+                            if (not excludewords(line)):
+                                result.append([idx, line])
+                                # remove element in original list
+                                keyword.remove(key)
+
+        elif (lv == 3):
+            for key in list(keyword):
+                if key in keyword:
+                    if re.search(key, line, re.IGNORECASE):
+                        reg = f'(?:(aplikasi)\s+[(umum)|(khusus)|(spbe)]?)'
+                        if re.search(reg, line, re.IGNORECASE):
+                            result.append([idx, line])
+                            keyword.remove(key)
+
+        elif (lv == 4):
+            res = [ele for ele in keyword if (ele in line)]
+            if (len(res) == len(keyword)):  # if all keyword is found in the same sentence
+                reg = f'(?:(aplikasi)\s+[(umum)|(khusus)|(spbe)]?)'
+                if re.search(reg, line, re.IGNORECASE):
+                    if (not excludewords(line)):
+                        result.append([idx, line])
+
+        else:
+            reg = f'(?:(aplikasi)\s+(spbe))'
+            if (re.search(reg, line.lower(), re.IGNORECASE)):
+                if (re.search(keyword[0], line.lower(), re.IGNORECASE)):
+                    if (not excludewords(line)):
```

Comment:
Please use the (`exclude_words`) function provided in `utility.py` to prevent redundancy